models/_sector/nft/trades/solana/nft_solana_trades_legacy.sql

```diff
@@ -0,0 +1,19 @@
+{{
+    config(
+        schema = 'nft_solana'
+        , tags = ['legacy']
+        , alias = alias('trades', legacy_model=True)
+        ,materialized = 'incremental'
+        ,file_format = 'delta'
+        ,incremental_strategy = 'merge'
+        ,unique_key = ['unique_trade_id','block_slot']
+        ,post_hook='{{ expose_spells(\'["solana"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["ilemi"]\') }}'
```

Comment:
You can remove these lines from the config of a dummy model. Then the model will default to a view and the CI error on spark should be resolved. (It was looking for the `unique_key` columns which aren't present in the dummy model).

Same comment applies to the other dummy models.

Reply:
done